docs/source/asr/speaker_diarization/intro.rst

```diff
@@ -0,0 +1,70 @@
+Speaker Diarization
+==================================
+
+Speaker Diarization (SD) is the task of segmenting audio recordings by speaker labels, that is Who Speaks When?
+
+.. image:: images/sd_pipeline.png
+        :align: center
+        :alt: Speaker Diarization pipeline [todo]
+        :scale: 50%
+
+
+A diarization system consists of **Voice Activity Detection (VAD)** model to get the time stamps of audio where speech is being spoken ignoring the background and 
+**Speaker Embeddings** model to get speaker embeddings on segments that were previously time stamped by VAD. 
```

Comment:
**Speaker Embeddings** model -> a **Speaker Embeddings** model
 on segments that were previously time stamped by VAD ->  on speech segments obtained  from VAD time stamps.